launcher/minecraft/mod/Mod.cpp

```diff
@@ -186,8 +186,11 @@ auto Mod::side() const -> QString
 
 auto Mod::mcVersions() const -> QString
 {
-    if (metadata())
-        return metadata()->mcVersions.join(", ");
+    if (metadata()){
+        QStringList reversed = metadata()->mcVersions;
+        std::reverse(reversed.begin(), reversed.end());
```

Comment:
I'm not sure this is the best place to put this.
A better place is where we set the information in the first place ([Packwiz.cpp](https://github.com/PrismLauncher/PrismLauncher/blob/a550277fda7c7c406a3b4159a87917bbdd243b2e/launcher/modplatform/packwiz/Packwiz.cpp))
Also, I know it may be a lot, but here it should correctly sort the version, as right now, if you have,
1.21.1, 1.21.2, and 1.21.10, the 1.21.10 will be in the middle regarding how you sort, as the sorting is made on strings